pkg/services/sqlstore/migrations/cloud_migrations.go

```diff
@@ -176,4 +176,9 @@ func addCloudMigrationsMigrations(mg *Migrator) {
 		Type:     DB_Text,
 		Nullable: true,
 	}))
+
+	mg.AddMigration("increase resource_uid column length", NewRawSQLMigration("").
```

Comment:
nit: add a comment explicitly stating we don't need to modify the column type in sqlite